core/templates/dev/head/pages/exploration_editor/editor_tab/state_editor_responses.html

```diff
@@ -99,10 +99,8 @@
       </button>
     </div>
   </md-card>
-  <md-card style="margin: 20px 0px; padding: 0px;" ng-if="!isCurrentInteractionTrainable()">
-{% else %}
-  <md-card style="margin: 0px; padding: 0px;">
 {% endif %}
+  <md-card style="margin: 0px; padding: 0px;">
```

Comment:
Should add newline above this, and deindent this and the following lines by 2?

Reply:
I think indentation is correct. The entire `if-else` is a part of a `<div>` tag. So everything in their body should have indentation?

Reply:
Oh, I see, sorry! In that case, what should happen is that the if-endif block from lines 94-102 should be indented by 2 more (it was wrong before). We optimize for the developer of the code here, not the final HTML generated.

Reply:
(It's optional for this to be fixed in this PR, since it's not a regression. I'm OK with the PR submitted as-is.)